docs/en/Running-Flink-on-Alluxio.md

```diff
@@ -98,3 +98,19 @@ Run the following command from the top level Flink project directory:
 {% include Running-Flink-on-Alluxio/wordcount.md %}
 
 Open your browser and check [http://localhost:19999/browse](http://localhost:19999/browse). There should be an output file `output` which contains the word counts of the file `LICENSE`.
+
+### Wordcount Example with Alluxio in fault tolerant mode
```

Comment:
this is repetitive of the previous section, could we instead make a note that flink users need to use commas in their URI authorities instead of semicolons?

I think we need more nuance to how we talk about `zk@` URIs in our docs. In many situations it's better to use config, so we don't want to make those users think that they need to use `zk@`. I suggest having a single place (perhaps HA doc) where we talk about the option of using `zk@` for URIs instead of configuring through site properties. Then for specific compute framework docs, we can specify whether commas or semicolons are needed, and link to the more detailed documentation for more details